src/github/mod.rs

```diff
@@ -289,6 +300,83 @@ pub fn existing_release_pr_location(
     None
 }
 
+pub fn get_commits_from_pr(
+    repo: &GithubRepo,
+    token: &str,
+    pr_number: &str,
+    client: &reqwest::Client,
+) -> String {
+    let pr_commits_url = format!("{}/pulls/{}/commits", repo.url, pr_number);
+
+    let mut res = match client
+        .get(&pr_commits_url)
+        .expect("Couldn't make a request builder for commits page url")
+        .header(UserAgent::new(USERAGENT.to_string()))
+        .header(Authorization(format!("token {}", token)))
+        .send()
+    {
+        Ok(response) => response,
+        Err(e) => {
+            panic!("Error in request to github for compare page: {}", e);
+        }
+    };
+    delay_if_running_out_of_requests(res.headers());
+
+    let mut buffer = String::new();
+    match res.read_to_string(&mut buffer) {
+        Ok(_) => (),
+        Err(e) => println!("error checking commit diff for {}: {}", pr_commits_url, e),
+    }
+
+    let prs: Vec<CommitInPR> = match serde_json::from_str(&buffer) {
+        Ok(v) => v,
+        Err(e) => panic!(format!("Couldn't deserialize repos from github: {}", e)),
+    };
+
+    let mut new_body = "automated release partay!\n\nPRs in this release:".to_string();
+
+    for c in prs.into_iter() {
```

Comment:
In the future, we'll want to extract a function to somewhere near here to enable the functionality requested in https://github.com/matthewkmayer/release-party-BR/issues/72 .